docs/custom_tokenizers.md

```diff
@@ -0,0 +1,49 @@
+# Custom Tokenizers
+
+HELM comes with many built-in tokenizers, but in some cases, you may need to add your own custom tokenizer for your custom model.
```

Comment:
The only question I have left after reading this is, in what cases do I need to add a custom tokenizer? Perhaps answering that question is out of the scope of this document/already answered elsewhere though.

Reply:
I think the implicit assumption here (you are trying to use a non-built-in tokenizer) is probably sufficient